test/parallel/test-path-parse-format.js

```diff
@@ -25,20 +25,27 @@ const assert = require('assert');
 const path = require('path');
 
 const winPaths = [
-  'C:\\path\\dir\\index.html',
-  'C:\\another_path\\DIR\\1\\2\\33\\\\index',
-  'another_path\\DIR with spaces\\1\\2\\33\\index',
-  '\\foo\\C:',
-  'file',
-  '.\\file',
-  'C:\\',
-  '',
+  // [path, root]
+  ['C:\\path\\dir\\index.html', 'C:\\'],
+  ['C:\\another_path\\DIR\\1\\2\\33\\\\index', 'C:\\'],
+  ['another_path\\DIR with spaces\\1\\2\\33\\index', ''],
+  ['\\foo\\C:', '\\'],
+  ['file', ''],
+  ['.\\file', ''],
+  ['C:', 'C:'],
+  ['C:.', 'C:'],
+  ['C:..', 'C:'],
+  ['C:abc', 'C:'],
+  ['C:\\', 'C:\\'],
+  ['C:\\abc', 'C:\\' ],
+  ['', ''],
```

Comment:
Could you add tests for NTFS alterate streams? e.g. `['cd:foo', '']`. Not just to test `root`, but also the other properties used in `checkParseFormat`.